site/_apps/cxjs-hackernews.txt

```diff
@@ -0,0 +1,27 @@
+---
+weight: 15
+title: 'CxJS Hacker News'
+github-title: 'codaxy/cxjs-hackernews'
+libraries: 
+    - name: 'CxJS'
+    - name: 'Preact'
+scaffolding: 'Cx CLI'
+module-bundling: 'Webpack'
+service-worker: 'Application Shell + runtime data caching with sw-precache (sw-precache-webpack-plugin)'
+perfomance-patterns: 'Code-splitting, CSS and JS inlining'
+server-side-rendering: 'None'
+api: 'Node-hnapi (unoffical)'
+hosting: 'Netlify'
+authors: 
+    - name: 'Marko Stijak'
```

Comment:
The name here will need to match the name in your `_contributors` submission in order to show you avatar underneath your app :)